getj2me

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+node --harmony ./_getj2me
```

Comment:
This is needed. See https://github.com/fxos/customizer/pull/11

I'll add a comment explaining it.